Foundation/NSTimeZone.swift

```diff
@@ -249,3 +292,7 @@ extension NSTimeZone {
     }
 
 }
+
+extension NSNotification.Name {
+    public static let NSSystemTimeZoneDidChange = NSNotification.Name(rawValue: "NSSystemTimeZoneDidChangeNotification") // NSUnimplemented
```

Comment:
@itaiferber, could you do me a favor and remove the raw value from this line? I think it's not supposed to be here (my mistake!). I'll create a PR to have it removed from master as well.